RecoTauTag/RecoTau/plugins/DeepTauId.cc

```diff
@@ -1812,11 +2033,14 @@ class DeepTauId : public deep_tau::DeepTauBase {
       get(dnn::pfCand_gamma_pvAssociationQuality) =
           getValueLinear<int>(candFunc::getPvAssocationQuality(gamma_cand), 0, 7, true);
       get(dnn::pfCand_gamma_fromPV) = getValueLinear<int>(candFunc::getFromPV(gamma_cand), 0, 3, true);
-      get(dnn::pfCand_gamma_puppiWeight) = getValue(candFunc::getPuppiWeight(gamma_cand));
-      get(dnn::pfCand_gamma_puppiWeightNoLep) = getValue(candFunc::getPuppiWeightNoLep(gamma_cand));
-      get(dnn::pfCand_gamma_lostInnerHits) = getValue<int>(candFunc::getLostInnerHits(gamma_cand, default_value));
+      get(dnn::pfCand_gamma_puppiWeight) = is_inner ? getValue(candFunc::getPuppiWeight(gamma_cand, 0.9084110f))
+                                                    : getValue(candFunc::getPuppiWeight(gamma_cand, 0.4211567f));
+      get(dnn::pfCand_gamma_puppiWeightNoLep) =
+          is_inner ? getValue(candFunc::getPuppiWeightNoLep(gamma_cand, 0.8857715703446466))
+                   : getValue(candFunc::getPuppiWeightNoLep(gamma_cand, 0.38226042182344516));
```

Comment:
the second argument should be a float literal, similar to `getPuppiWeight` calls